src/openrct2/platform/shared.c

```diff
@@ -509,10 +510,28 @@ void platform_process_messages()
 				int gesturePixels = (int)(_gestureRadius * gScreenWidth);
 				if (gesturePixels > tolerance) {
 					_gestureRadius = 0;
-					keyboard_shortcut_handle_command(SHORTCUT_ZOOM_VIEW_IN);
+					
+					if (gScreenFlags & SCREEN_FLAGS_TITLE_DEMO)
+						return;
```

Comment:
That makes sense. I had just copied this here from elsewhere, and wasn't expecting to leave it here (though I'm not sure where best to place it). I'll change it to break, but I'll have to remember to change it back to return if I do end up moving it.